src/core/RecyclerListView.js

```diff
@@ -467,6 +466,10 @@ RecyclerListView.propTypes = {
     //On scroll callback onScroll(rawEvent, offsetX, offsetY), note you get offsets no need to read scrollTop/scrollLeft
     onScroll: PropTypes.func,
 
+    //Provide your own ScrollView Component. The contract for the scroll event should match the native scroll event contract, i.e.
+    // event = { nativeEvent: { contentOffset: { x: offset, y: offset } } }
+    externalScrollViewer: PropTypes.func,
```

Comment:
call it externalScrollView and also publish the full contract. I think we need on sizeChanged too and onScroll event